### PR TITLE
update docs to mention spread algorithm

### PR DESCRIPTION
**Pull request by shantanugadgil**
my attempt to update docs to mention `spread` algorithm.
(do clean up/re-arrange the sentence as you feel appropriate) 😃 

@tgross @angrycub

### CODE REVIEW

File: website/pages/docs/configuration/server.mdx [hunk 1]
```diff
@@ -241,6 +241,9 @@ server {
 
 ### Configuring Scheduler Config
 
+- `scheduler_algorithm` `(string: "binpack")` - Specifies the schedulaer algorithm to place
```
Comment (tgross) [inline website/pages/docs/configuration/server.mdx:244]:
```suggestion
- `scheduler_algorithm` `(string: "binpack")` - Specifies the scheduler algorithm to place
```

Comment (shantanugadgil) [reply website/pages/docs/configuration/server.mdx]: fixed

**Comment by angrycub** [comment]
@shantanugadgil - I very much appreciate this PR, because it's not immediately apparent how to configure spread scheduling.

I'm concerned about documenting this here rather than making the canonical documentation at https://www.nomadproject.io/api-docs/operator/#update-scheduler-configuration more apparent.  This configuration scheme only works when bootstrapping the cluster, so linking to the API page actually makes the (imo) more frequently necessary configuration route more visible.  If we are going to double the documentation, then we need to document all of the attributes, not just scheduling_algorithm; but, again, I am reluctant to double document for concern about keeping them in sync long term.

**Comment by shantanugadgil** [comment]
Not having to repeat the docs is correct, I agree, but as an operator (not usually bothered with making HTTP API calls), having the docs mention _this_ up front would be better.
This will let users know what is possible _up front_.

Also, as an operator, I am more concerned what are the configuration parameters via **configuration files**.

I do agree, that everything need not be repeated, but a placeholder can be mentioned in the configuration section for all the **configuration knobs** that exist.

I stumbled upon this in the following way: changelog -> ticket -> url to HTTP API .

My usual method is:
1. google search for 'nomad stanza server' -> click url
2. I have all my 'most common' settings.

**Comment by angrycub** [comment]
I agree. I've got some suggested prose that I'm testing locally right now that I'd like to run by you shortly. My main worry here is that this is a *bootstrap-only* config.  So that's what's leading me to not fully documenting it here.  I 100% agree that this section would benefit from the words "spread scheduling" making an appearance though to make it more visible to search engines

**Comment by angrycub** [comment]
I've committed an alternate treatment.  We can refine or revert from there.  My goal in that version was to:

- bring in "spread scheduling" as a phrase present on that page for visibility to the search engines
- highlight that that configuration setting is bootstrap only
- aim people at the canonical docs in the API section both for visibility should they need to change it in a running cluster
- reduce sources of truth in the docs

Thanks again for your feedback and collaboration on this one!

**Comment by angrycub** [comment]
@shantanugadgil what do you think?

**Comment by shantanugadgil** [comment]
It is ok to drop this PR.

> I've committed an alternate treatment. We can refine or revert from there. My goal in that version was to:

I got multiple notifications and am confused what to review/look at?

> * bring in "spread scheduling" as a phrase present on that page for visibility to the search engines

Yes, this seems fine!

> * highlight that that configuration setting is bootstrap only

The obvious next question is ... can I change this later and then do doing rolling restarts of servers?
So something like "if you change this value you will have to restart all servers"
(also, then what happens when some 💡 forgets to restart a server or two? how can check using command line, the currently active strategy in the cluster?)

> * aim people at the canonical docs in the API section both for documentation and visibility should they need to change it in a running cluster
> * reduce sources of truth in the docs

This, I am not convinced about, but I leave the call upto you folks. 💯 
In the spirit of being operator-friendly, pointing them to HTTP API docs is _ok_, but not great.
(of course, this is just my opinion) 😃

**Comment by shantanugadgil** [comment]
I read the changes. They seem fine to me except the "bootstrap" related questions from my previous post: https://github.com/hashicorp/nomad/pull/7994#issuecomment-630363042

**Comment by shantanugadgil** [comment]
> 
> 
> LGTM.
> 
> There's a merge conflict left @angrycub but once that's resolve this looks 👍

my first attempt at resolving merges in the web editor! 👀

**Comment by angrycub** [comment]
Go team, go!  Thanks for the help and extra eyes, @shantanugadgil

**Comment by cgbaker** [comment]
cherry-picked to release-0.11.3

**Pull request merged**